Explorer/Assets/DCL/SDKComponents/AudioStream/AudioStreamSystem.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using DCL.Diagnostics;
+using DCL.ECSComponents;
+using DCL.Optimization.Pools;
+using DCL.Utilities.Extensions;
+using ECS.Abstract;
+using ECS.Unity.Groups;
+using SceneRunner.Scene;
+
+namespace DCL.SDKComponents.AudioStream
+{
+    [UpdateInGroup(typeof(ComponentInstantiationGroup))]
+    [LogCategory(ReportCategory.AUDIO_STREAM)]
+    public partial class AudioStreamSystem : BaseUnityLoopSystem
```

Comment:
There is no handling for `PBAudioStream` Component removal. `ReleasePoolableComponentSystem<T, TProvider>` handles entity removal/world destruction only! In [wiki it's described](https://github.com/decentraland/unity-explorer/wiki/How-To#generalized-scenarios) with details